### PR TITLE
added content on dashboard alerts

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-8379

### CODE REVIEW

**Review by quchangl-github — APPROVED** [review]

File: observability/design_grafana.adoc [hunk 1]
```diff
@@ -1,7 +1,11 @@
 [#using-grafana-dashboards]
 = Using Grafana dashboards
 
-Use Grafana dashboards to view hub cluster and managed cluster metrics.
+Use Grafana dashboards to view hub cluster and managed cluster metrics. The data displayed in the Grafana alerts dashboard relies on `alerts` metrics, originating from managed clusters. The `alerts` metric has nothing to do with managed clusters forwarding alerts to {prod-short} alert manager on the hub cluster. Therefore, the metrics and alerts have distinct propagation mechanisms and follow separate code paths.
```
Comment (dockerymick) [inline observability/design_grafana.adoc]: The correct conref for the product is `{product-title-short}`. The conrefs are listed in any of the `common-attributes.adoc` files in the rhacm-docs repo.

```suggestion
Use Grafana dashboards to view hub cluster and managed cluster metrics. The data displayed in the Grafana alerts dashboard relies on `alerts` metrics, originating from managed clusters. The `alerts` metric does not affect managed clusters forwarding alerts to {product-title-short} alert manager on the hub cluster. Therefore, the metrics and alerts have distinct propagation mechanisms and follow separate code paths. 
```

File: observability/design_grafana.adoc [hunk 2]
```diff
@@ -1,7 +1,11 @@
 [#using-grafana-dashboards]
 = Using Grafana dashboards
 
-Use Grafana dashboards to view hub cluster and managed cluster metrics.
+Use Grafana dashboards to view hub cluster and managed cluster metrics. The data displayed in the Grafana alerts dashboard relies on `alerts` metrics, originating from managed clusters. The `alerts` metric has nothing to do with managed clusters forwarding alerts to {prod-short} alert manager on the hub cluster. Therefore, the metrics and alerts have distinct propagation mechanisms and follow separate code paths. 
+
+Even if you see data in the Grafana alerts dashboard, that does not guarantee that the managed cluster alerts are successfully forwarding to the {prod-short} hub cluster alert manager. If the metrics are propagated from the managed clusters, you can view the data displayed in the Grafana alerts dashboard.
```
Comment (dockerymick) [inline observability/design_grafana.adoc]:
```suggestion
Even if you see data in the Grafana alerts dashboard, that does not guarantee that the managed cluster alerts are successfully forwarding to the {product-title-short} hub cluster alert manager. If the metrics are propagated from the managed clusters, you can view the data displayed in the Grafana alerts dashboard. 
```

**Review by dockerymick — CHANGES_REQUESTED** [review]

**Review by dockerymick — APPROVED** [review]
Thanks for the updates Jake. this looks good to me!

**Pull request merged**